Library/Homebrew/test/rubocops/patches_cop_spec.rb

```diff
@@ -84,6 +85,15 @@ def patches
                                   line: 5,
                                   column: 5,
                                   source: source }]
+        elsif patch_url =~ %r{https?://patch-diff\.githubusercontent\.com/raw/(.+)/(.+)/pull/(.+)\.(?:diff|patch)}
+          expected_offenses = [{  message:  "use GitHub pull request URLs:\n"\
```

Comment:
Favour using `EOS.indent` over embedding `\n` in multiline strings. If you've done this in other tests: make a new PR (or add to this one) that global style change. Alternatively, these could all be addressed in another PR after this is merged?